### DISCUSSION TITLE
Request: Indicate playback speed when in ffwd/rewind mode

### DISCUSSION THREAD

**Discussion by gkertai** [videovillage/community#126] [Ideas]
QuickTime Player does a decent job of this, by showing 2×, 5×, 10×, 30× on the control panel.

![Screen Shot 2020-01-29 at 21 27 53](https://user-images.githubusercontent.com/1874498/73394740-71583300-42de-11ea-83fd-6be0357fa901.PNG)

Screen has a more minimalistic control panel, yet there's room for such info when *Show Controls* is active. A display of ⏪/⏩ (rewind/forward) icons would also be nice to show current playback direction. In *Hide Controls* mode a brief pop-up overlay on the video could indicate changing the playback speed/direction.

This would help the user when having to skim through unchanging content (e.g. black or color bars). It would also help confirm that the proper keyboard shortcut was used.

On a somewhat related note: a pop-up overlay to indicate number of frames/seconds skipped would also be nice when you use the Step/Short Step/Long Step keyboard shortcuts. E.g. "+10" to indicate ten frames forward, or "-2s" to indicate 2 seconds backwards.

**Comment by gregcotten** [comment]
A while ago I gave some thought to where such info should display even when "Show Controls" is active - I'll have another look!
